Cargo.toml

```
@@ -23,6 +23,7 @@ subtle = { version = "2.1.1", default-features = false }
 digest = { version = "0.10.0", default-features = false, features = ["alloc"] }
 pkcs1 = { version = "0.4", default-features = false, features = ["pkcs8", "alloc"] }
 pkcs8 = { version = "0.9", default-features = false, features = ["alloc"] }
+signature = { version = "1.5", default-features = false , features = ["rand-preview"] }
```

To keep the `rand_core` versions properly pinned you need to do:

```suggestion
signature = { version = ">=1.5, <1.7", default-features = false , features = ["rand-preview"] }
```

---

Done